### PR TITLE
Grammar fixes in smart contracts developer doc

### PR DESCRIPTION
**Pull request by naveen106**
Some grammatical fixes.

## Description

Grammatical fixes in some sentences.

## Related Issue
grammatical changes.

### CODE REVIEW

File: src/content/developers/docs/smart-contracts/index.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Like how a vending machine removes the need for a vendor employee, smart contrac
 
 ## Permissionless {#permissionless}
 
-Anyone can write a smart contract and deploy it to the network. You just need to learn how to code in a [smart contract language](/developers/docs/smart-contracts/languages/), and have enough ETH to deploy your contract. Deploying a smart contract is technically a transaction, so you need to pay your [Gas](/developers/docs/gas/) in the same way that you need to pay gas for a simple ETH transfer. Gas costs for contract deployment are far higher, however.
+Anyone can write a smart contract and deploy it to the network. You just need to learn how to code in [smart contract language](/developers/docs/smart-contracts/languages/), and have enough ETH to deploy your contract. Deploying a smart contract is technically a transaction, so you need to pay [Gas](/developers/docs/gas/) in the same way you need to pay gas for a simple ETH transfer. However, gas costs for contract deployment are far higher.
```
Comment (minimalsm) [inline src/content/developers/docs/smart-contracts/index.md:71]: We should keep 'a' before smart contract language (there are multiple)

```suggestion
Anyone can write a smart contract and deploy it to the network. You just need to learn how to code in a [smart contract language](/developers/docs/smart-contracts/languages/), and have enough ETH to deploy your contract. Deploying a smart contract is technically a transaction, so you need to pay [Gas](/developers/docs/gas/) in the same way you need to pay gas for a simple ETH transfer. However, gas costs for contract deployment are far higher.
```

File: src/content/developers/docs/smart-contracts/index.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ Another limitation of smart contracts is the maximum contract size. A smart cont
...
+Multisig (multiple-signature) contracts are smart contract accounts that require multiple valid signatures to execute a transaction. This is very useful for avoiding single points of failure for contracts holding substantial amounts of ether or other tokens. Multisigs also divide responsibility for contract execution and key management between multiple parties and prevent the loss of a single private key leading to irreversible loss of funds. For these reasons, multisig contracts can be used for simple DAO governance. Multisigs require N signatures out of M possible acceptable signatures (where N ≤ M, and M > 1) in order to execute. `N = 3, M = 5` and `N = 4, M = 7` are commonly used. A 4/7 multisig requires four out of seven possible valid signatures. This means the funds are still retrievable even if three signatures are lost. In this case it also means that majority of key-holders must agree and sign in order for the contract to execute.
```
Comment (minimalsm) [inline src/content/developers/docs/smart-contracts/index.md:98]: Grammar change doesn't make sense. This is better:

```suggestion
Multisig (multiple-signature) contracts are smart contract accounts that require multiple valid signatures to execute a transaction. This is very useful for avoiding single points of failure for contracts holding substantial amounts of ether or other tokens. Multisigs also divide responsibility for contract execution and key management between multiple parties and prevent the loss of a single private key leading to irreversible loss of funds. For these reasons, multisig contracts can be used for simple DAO governance. Multisigs require N signatures out of M possible acceptable signatures (where N ≤ M, and M > 1) in order to execute. `N = 3, M = 5` and `N = 4, M = 7` are commonly used. A 4/7 multisig requires four out of seven possible valid signatures. This means the funds are still retrievable even if three signatures are lost. In this case, it also means that the majority of key-holders must agree and sign in order for the contract to execute.
```

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
Thanks for the PR @naveen106 

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by naveen106** [comment]
Thanks @minimalsm Pleased to contribute in this growing community of ethereum

**Pull request merged**